core/standards.md

```diff
@@ -0,0 +1,93 @@
+# Core Smart Contracts
+
+## Definition of Core
+
+A core contract is one that meets any one of the following:
+
+1. Any contract that is part of the [dss repo](https://github.com/makerdao/dss)
+2. Will have ward status on or other privileged access to any other core contract (i.e. `Vat.rely(<new contract address>)`)
+3. Will rely on the Pause Proxy as a permissioned actor. If the pause proxy will at any point make an external call to this contract then there is additional security and process work that needs to be done.
+4. Will be included in the chainlog as a canonical “Maker Contract”. Even if the contract does not have elevated permissions in the system, a failure of a contract in the chainlog could result in cost (bug bounties) or reputation damage for the protocol.
+5. If a contract is put forward to a business partner of the DAO, by DAO members, close consideration should be given to whether it would be perceived as core Maker brand by those partners.
+
+## Protocol Engineering Review Process for Core Contracts
+
+All Core contracts need to go through a risk adjusted review process.  Risk should be evaluated at the beginning of the project and continue to be re-evaluated through the engineering process.
+
+Our review process at a minimum requires two team member reviews and approvals.  However for higher risk contracts may also involve Red Team reviews and audits by external auditors.
+
+Core contracts should follow established coding patterns including repo and contract structures, variable naming patterns (yes [Daiwanese](https://docs.makerdao.com/other-documentation/system-glossary) is a standard for our team) and auth patterns. For most contracts, compiler version should be fixed at 0.8.n or 0.6.12 and optimizations should be off.  While licensing is considered on a case by case basis, it should always be declared and we tend to default to AGPLv3. We value security over gas optimizations and simplicity over complexity. As such, low level calls should be used minimally and gas optimization tricks should be used sparingly.
```

Comment:
Additional comment here, maybe we should tell devs **not** to just hop 0.5.12 to 0.8.14 without a documented reason why? We would be documenting our stuff as we upgrade so might be a good idea

Reply:
I think this is a good point but doesn't feel like the right place to add it. It is more about maintenance and upgrading/changes.  I'll add something to the documentation for this i think

Reply:
Ok cool